build.js

```diff
@@ -22,13 +23,13 @@ await build({
   plugins: [
     {
       name: 'alias',
-      async setup({ onResolve, resolve }) {
+      async setup({ onLoad, onResolve, resolve }) {
         const stubFiles = await readdir('src/stubs', { withFileTypes: true });
         // These packages are imported, but can be stubbed.
         const stubNames = stubFiles
           .filter((file) => file.isFile())
           .map((file) => parse(file.name).name);
-        onResolve({ filter: new RegExp(`^(${stubNames.join('|')})$`) }, ({ path }) => ({
+        onResolve({ filter: new RegExp(`^(${stubNames.join('|')})(\\/|$)`) }, ({ path }) => ({
```

Comment:
when `utils` would be a stubname,
not only
`import "utils"`
and
`import "utils/"`
but also
`import "utils/foo/bar/bung/baz"`
would be matched. is this wanted?
dont you mean `/^(utils)\/?$/`

Reply:
You’re right. This shouldn’t have changed. I reverted it.